rest_sensors.cpp

```diff
@@ -785,6 +785,10 @@ int DeRestPluginPrivate::changeSensorConfig(const ApiRequest &req, ApiResponse &
                         rsp.list.append(errorToMap(ERR_INTERNAL_ERROR, QString("/sensors/%1").arg(id), QString("Internal error, %1").arg(ERR_BRIDGE_BUSY)));
                     }
                 }
+                //don't update value for those setting, let them be filled by the return from device
+                else if ( (rid.suffix == RConfigTempThreshold) or (rid.suffix == RConfigHumiThreshold))
```

Comment:
This isn't C++, it shouldn't even compile?
Please only submit PR's after they have been compiled locally without warnings and errors.

Reply:
also white space, parentheses

```
else if (rid.suffix == RConfigTempThreshold || rid.suffix == RConfigHumiThreshold)
{
}
```

Reply:
I have tested again, it compile without problem on raspbian ^^
But not sure for result.